tests/integration/fileTests.py

```diff
@@ -122,23 +120,24 @@ def test_file_submission(self):
 
         # Test that job ids are returned
         responseDict = json
-        fileKeys = ["program_activity", "award_financial",
-            "appropriations"]
-        for key in fileKeys:
-            idKey = "".join([key,"_id"])
-            self.assertIn(idKey, responseDict)
-            jobId = responseDict[idKey]
-            self.assertIsInstance(jobId, int)
-            # Check that original filenames were stored in DB
-            originalFilename = self.interfaces.jobDb.getOriginalFilenameById(jobId)
-            self.assertEquals(originalFilename,self.filenames[key])
-        # check that submission got mapped to the correct user
-        submissionId = responseDict["submission_id"]
-        self.file_submission_id = submissionId
-        submission = self.interfaces.jobDb.getSubmissionById(submissionId)
+        fileKeys = ["program_activity", "award_financial", "appropriations"]
+        with createApp().app_context():
```

Comment:
I'm not sure how to do this, but there's probably a way to call this within the class's `setUp` and `tearDown` functions (or using a py.test fixture) rather than in the various tests. It's context that kind of muddies up individual tests